### DISCUSSION TITLE
buddhism/2023/origination/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#110] [General]
# buddhism/2023/origination/

https://shanenull.com/buddhism/2023/origination/

**Comment by shane0** [comment]
From heart sutra commentary book

practice [to break the links]. The forward way of perceiving purity is based on the severance of the fundamental hindrances, while the reverse way is based on repeated contemplation after severance.
The forward way of perceiving defilement is, first, to perceive that the substantial nature lof life] contains twelve links (nidanas):
(1) ignorance (avidya), meaning to be deluded about the internal and foolish (yu) about the external; (2) karmic conditioning (samskara), which means [moving on to] fortunate or unfortunate [circumstances or rebirths] or not moving [up or down in the next life]; (3) consciousness (vijñana) means the karmic-maturing (vipaka) consciousness; (4) name and form (namarupa), meaning the five skandhas; (5) six sense spheres (sadayatana), meaning the six sense organs; (6) sensory contact (sparsa), meaning contact with objects; (7) feelings (vedana), meaning painful and pleasurable sense objects; (8) desire (trsna), referring to clinging to the three realms; (9) appropriation (upadana), meaning afflictions (klesas); (10) becoming (bhava), meaning the seeds —[developed] from "karmic conditioning" up to "feeling" — moistened by desire and appropriation, [engendering fruit that] leads from one existence to another; (11) birth (jati), the reappearance of suffering resulting [from previous existence]; and (12) old age and death (jara-marana), that is, [the physical changing, weakening, and ending of hife.
Next, one should contemplate that erroneous actions produced from ignorance can result in the consciousness that manifests [seeds of] past karma. Then, the characteristics of the five skandhas appear and the sense organs develop. After contacting and feeling the field of objects of cognition (visaya-gocara), the
"seeds" establish results. Indulging in desires, one increases klesas that in turn "moisten" previous karma. [Consequently] the suffering of the five destinies (gati) — birth, old age, death, sorrow, and misery — arise. Therefore, the sutra says ignorance conditions samskaras and so forth, until we come to birth conditioning old age and death. The five links of consciousness and so forth are not necessarily in sequence and, therefore, according to whatever

**Reply by shane0** [reply]
Page 107

**Reply by shane0** [reply]
https://github.com/shane0/buddhism/discussions/82#discussioncomment-8235167

**Comment by shane0** [comment]
# 12 death

https://shanenull.com/buddhism/2023/maranasati/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2024/heart_sutra/
